dffml/util/data.py

```diff
@@ -109,13 +131,45 @@ def traverse_get(target, *args):
     >>>
     >>> traverse_get({"one": {"two": 3}}, "one", "two")
     3
+    >>> traverse_get({"one": {"two": 3}}, "one.two")
+    3
+    >>> traverse_get({"one.two": {"three": 4}}, "'one.two'.three")
+    4
     """
+    if len(args) == 1:
+        args = split_dot_seperated(args[0])
     current = target
     for level in args:
         current = current[level]
     return current
 
 
+def traverse_set(target, *args, value):
+    """
+    Examples
+    --------
+
+    >>> from dffml import traverse_set
+    >>>
+    >>> d = {"one": {"two": 3}}
+    >>> traverse_set(d,"one.two", value = "Three")
+    >>> d["one"]["two"]
+    'Three'
+    """
+    if len(args) == 1:
+        args = split_dot_seperated(args[0])
+    if len(args) == 1:
+        target[args[0]] = value
+        return
+
+    current = target
+    for level in args[:-1]:
+        current = current[level]
```

Comment:
This might not be correct, but the point it we need to be able to create dict's if they don't exit on the way down, see https://github.com/intel/dffml/pull/694/files#r437573092

```suggestion
        if not level in current:
            current[level] = {}
        current = current[level]
```